dvc/objects/checkout.py

```diff
@@ -212,6 +128,24 @@ def _diff(
     return diff
 
 
+class Link:
+    def __init__(self, links):
+        self._links = links
```

Comment:
What this really is is transfer with a config. Will be moved over after `meta` support in `odiff`.